config/environments/production.rb

```diff
@@ -56,7 +56,11 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+  Rails.logger = Logger.new(STDOUT)
+  Rails.logger.level = Logger::INFO
```

Comment:
Shouldn't this be `WARN`? I actually don't understand why rails would make you specify the log level twice (once in `Rails.logger.level` and once in `config.log_level`) but it may just be my lack of familiarity with rails.

Reply:
Hi @timwis  regarding your comment, you are right, I did set it to "WARN" at line 63. so we would not print anything below WARN in production.

Line 60 defines the default level for rails messages, I was referencing this guide: https://stackify.com/rails-logger-and-rails-logging-best-practices/